internal/ignition/ignition_test.go

```diff
@@ -1413,14 +1415,23 @@ var _ = Describe("IgnitionBuilder", func() {
 	})
 
 	It("no multipath for okd - config setting", func() {
-		config := IgnitionConfig{OKDRPMsImage: "image"}
+		ignitionConfig.OKDRPMsImage = "image"
 		mockMirrorRegistriesConfigBuilder.EXPECT().IsMirrorRegistriesConfigured().Return(false).Times(1)
-		text, err := builder.FormatDiscoveryIgnitionFile(context.Background(), &infraEnv, config, false, auth.TypeRHSSO, "")
+		text, err := builder.FormatDiscoveryIgnitionFile(context.Background(), &infraEnv, ignitionConfig, false, auth.TypeRHSSO, "")
 
 		Expect(err).Should(BeNil())
 		Expect(text).ShouldNot(ContainSubstring("multipathd"))
 	})
 
+	It("okd support disabled", func() {
+		ignitionConfig.EnableOKDSupport = false
```

Comment:
Why `ignitionConfig` is being reused across tests? If this test is moved up this will break other tests.

Lets create `ignitionConfig` and `okdIgnitionConfig` as globals instead

Reply:
ignitionConfig it's setup in a BeforeEach condition: AFAIU each spec this should be initialised again, effectively not changing the current behaviour. Being centralized it's easier to define default params (simulating env var default values instead of native default values).

Reply:
ah, I see, okay